include/tnn_api.h

```diff
@@ -563,6 +545,27 @@ template <typename T, int WIDTH> class NetworkModule : public Module {
         max_batch_size_ = 0;
     }
 
+    void printDeviceMatrix(const DeviceMatricesView<T> &device_matrices_view, int line_break_every) {
```

Comment:
redundant now. You have the same function in the DeviceMatricesView.

Reply:
move to io/result_check, consolidate with other devicematrix